api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -42,6 +44,7 @@ import java.util.concurrent.ExecutorService
 
 @Suppress("unused", "UNUSED_VARIABLE", "EmptyFunctionBlock", "DEPRECATION")
 private class PurchasesAPI {
+    @OptIn(InternalRevenueCatAPI::class)
```

Comment:
Is this the equivalent to what we have in iOS with `_spi(Internal)`. Just curious :)

Reply:
Yup! You need to explicitly annotate usages of these APIs like this.